karrot/applications/templates/application_accepted.mjml

```diff
@@ -9,6 +9,7 @@
             <mj-column>
                 <mj-text>
                     <h1>{% trans group_name=group.name %}Your application to join {{ group_name }} was accepted{% endtrans %}</h1>
+                    <p> {{ group.welcome_message }} </p>
```

Comment:
May need to wrap this into a `if` block, as we don't want to have an empty `p` tag if the group didn't specify a welcome message.

Reply:
Good point. `{% if group.welcome_message %}<p> {{ group.welcome_message }} </p>{% endif %}` should cover both when `welcome_message` is `False` and `""` if I'm not mistaken.

Reply:
The case this doesn't cover would be if the field has only whitespace in it. Not sure if that's enough of a problem to handle - and if so, should it be handled here or as a check when you save the field?

Reply:
Good catch! I would ignore the "only whitespace" case for now, we also don't handle it for other fields and it doesn't seem to be a problem.

Reply:
Sounds good. Anything else you want me to handle in this PR? I'm thinking about the invitation email, or joining an open group. Otherwise I think it should be ok to merge.